File: Client/FinanceLib/Models/Wallet.cs
namespace FinanceLib.Models
{
    public class Wallet
    {
        // IBAN code of the wallet
        private string iban;
        public string IBAN
        {
            get { return iban; }
            set { iban = value; }
        }
        
        // Cash inside the wallet
        private double cash;
        public double Cash
        {
            get { return cash; }
            set { cash = value; }
        }
    }
}

File: Server/financelib/wallet.py
from random import randint

class Wallet:
    def __init__(self):
        self._iban = str(randint(1, 1000))
        self._cash = randint(0, 100000)
    
    @property
    def iban(self):
        return self._iban
    
    @property
    def cash(self):
        return self._cash
    
    @cash.setter
    def cash(self, value):
        self._cash = value
    
    def to_json(self):
        return {
            'iban': self.iban,
            'cash': self.cash,
        }